app/src/main/java/org/mifos/mobile/ui/fragments/RegistrationFragment.kt

```diff
@@ -4,6 +4,7 @@ import android.graphics.PorterDuff
 import android.os.Bundle
 import android.text.Editable
 import android.text.TextWatcher
+import android.util.Log
```

Comment:
@ritish099 I think this is unused import. Have you verified with './gradlew check' ?

Reply:
@PatelVatsalB21 thanks for pointing out..